README.md

```diff
@@ -24,10 +25,22 @@ with `rustup`:
 $ rustup component add llvm-tools-preview
 ```
 
-For BOLT, it's unfortunately more complicated. See [below](#bolt-installation) for BOLT installation
-guide.
+For BOLT, it is highly recommended to use [Docker](#docker).
+See [below](#bolt-installation) for BOLT installation guide.
+
+> BOLT and Docker support is currently *experimental*.
+
+## Docker
+
+`cargo-pgo` has builds deployed to Docker Hub, to run `cargo-pgo` on any system that has Docker, run this in your project directory:
```

Comment:
Let's just add instructions for `docker build -t cargo-pgo .` for now, I'll modify the README once I add some Docker registry later.

Reply:
Got that! I'll finish it ASAP.